app/admin/resources/dashboard.rb

```diff
@@ -21,7 +21,36 @@
     #       end
     #     end
     #   end
+    columns do
```

Comment:
I think we can merge it. But we should also create an issue to add the translations.
You can either do both or delegate to me for the latter.
bussi :kiss: